### PR TITLE
Update license to include Apache 2.0

### PR DESCRIPTION
**Pull request by skim1420**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,3 +1,208 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright 2014 Netflix
```
Comment (davidchambers) [inline LICENSE]: Netflix?

Comment (skim1420) [inline LICENSE]: Sorry, inadvertently copied from another project. I'll rebase and squash.

**Comment by davidchambers** [comment]
Let's also replace the deprecated `"licenses"` field in __package.json__ with a [`"license"`][1] field:

    "license": "(Apache-2.0 OR WTFPL)"


[1]: https://docs.npmjs.com/files/package.json#license

File: package.json [hunk 2]
```diff
@@ -4,12 +4,7 @@
   "description": "Base64 encoding and decoding",
   "author": "David Chambers <dc@davidchambers.me>",
   "main": "./base64.js",
-  "licenses": [
-    {
-      "type": "WTFPL",
-      "url": "https://raw.github.com/davidchambers/Base64.js/master/LICENSE"
-    }
-  ],
+  "license": "(Apache-2.0 OR WTFPL)"
```
Comment (davidchambers) [inline package.json]: We're missing a comma at the end of this line.

File: LICENSE [hunk 3]
```diff
@@ -1,3 +1,206 @@
...
+
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
```
Comment (davidchambers) [inline LICENSE]: We're missing this line, I think:

    Copyright [yyyy] [name of copyright owner]

It should be something like:

    Copyright 2011..2015 David Chambers

**Comment by skim1420** [comment]
Sorry about the sloppiness, I think my latest commit addresses these.

**Comment by davidchambers** [comment]
> Sorry about the sloppiness, I think my latest commit addresses these.

No worries, and thank you.

I fixed the Travis failure with #21, so if you rebase the tests should pass. Please squash your commits while you're at it.

**Comment by skim1420** [comment]
Squashed commits.

**Pull request merged**